target/linux/ramips/mt7621/base-files/etc/board.d/01_leds

```diff
@@ -43,46 +43,46 @@ mikrotik,rbm11g)
 	ucidef_set_led_rssi "rssiveryhigh" "RSSIVERYHIGH" "$boardname:green:rssi4" "wlan0" "80" "100"
 	;;
 mtc,wr1201)
-	ucidef_set_led_switch "eth_link" "LAN link" "$boardname:green:eth_link" "switch0" "0x0f"
+	ucidef_set_led_netdev "eth_link" "LAN link" "$boardname:green:eth_link" "br-lan"
```

Comment:
@adrianschmutzler @981213 
As `netdev` trigger does not support multiple interfaces, I have to use `br-lan` here. Any better ideas?